src/equations/equations.jl

```diff
@@ -326,6 +326,18 @@ combined with certain solvers (e.g., subcell limiting).
 """
 function n_nonconservative_terms end
 
+"""
+	has_antisymmetric_flux(flux,equations)
```

Comment:
We should try to find a better name. If I understood correctly, the structure you need is that